src/Catalog/Helpers/IGalleryDatabaseQueryService.cs

```diff
@@ -9,6 +9,7 @@ namespace NuGet.Services.Metadata.Catalog.Helpers
 {
     public interface IGalleryDatabaseQueryService
     {
+        Task<FeedPackageDetails> GetPackage(string id, string version);
```

Comment:
>GetPackage [](start = 33, length = 10)

What do you think of the name `GetPackageOrNull` to clarify the 'not found' behavior?

Reply:
Better! Fixed in https://github.com/NuGet/NuGet.Services.Metadata/pull/537/commits/6e19b926fed8908b24abe4f0597486073991c65c